endpoints.json

```diff
@@ -12,5 +12,13 @@
     "status": "active",
     "version": "0.1.0",
     "URL": "https://nuts-fhir.eu.ngrok.io/t/fhir/"
+  },
+  {
+    "endpointType": "urn:nuts:endpoint:consent",
+    "identifier": "urn:ietf:rfc:
+    1779:O=Nuts,C=NL,L=Groenlo,CN=nuts_corda_development_CHANGE_ME",
+    "status": "active",
+    "version": "0.1.0",
+    "URL": "tcp://1.tcp.eu.ngrok.io:CHANGE_ME"
```

Comment:
Dear CHANGE_ME, there are limits to the use of CHANGE_ME, it is not a valid port number ;)

Reply:
^^